torchvision/models/__init__.py

```diff
@@ -14,4 +14,4 @@
 from .vision_transformer import *
 from .swin_transformer import *
 from . import detection, optical_flow, quantization, segmentation, video
-from ._api import get_weight
+from ._api import get_weight, list_models, load
```

Comment:
We might want to call this `load_model` (and rename `datasets.load` into `load_dataset` as well) to avoid conflicts?

Reply:
Yeah sounds good to me. I'm not going to rename load for datasets on this PR (cause it's going to add too many changes) but I'll rename `model_load()`